pkg/hub/controllers/workerslicegwrecycler/utils.go

```diff
@@ -241,10 +252,10 @@ func (r *Reconciler) update_routing_table(e *fsm.Event) error {
 			return errors.New("route not yet present")
 		}
 		return nil
-	}, retry.Attempts(100))
+	}, retry.Attempts(200))
 
 	if err != nil {
-		log.Error(err, "Err():", err.Error())
+		log.Error(err, "Err(): moving FSM to error state:", err.Error())
```

Comment:
This should be - `log.Error(err, "Err(): moving FSM to error state:") `in the end it expects key/values